lib/elixir/lib/integer.ex

```diff
@@ -400,17 +400,23 @@ defmodule Integer do
       iex> Integer.gcd(10, 0)
       10
 
+      iex> Integer.gcd(7, 7)
+      7
+
       iex> Integer.gcd(0, 0)
       0
 
   """
   @spec gcd(integer, integer) :: pos_integer
+  @spec gcd(int, int) :: int when int: pos_integer
+  @spec gcd(0, 0) :: 0
```

Comment:
It is defined to be 0 mathematically by convention, I contributed a doc patch today to clarify this point https://github.com/elixir-lang/elixir/pull/5944.